src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/AggregatedRssFeedProducer.java

```diff
@@ -64,31 +70,40 @@
      * @param limit Upper limit of RSS entries count in the generated feed. If equal or less
      *     than zero then all approved blog posts will be generated.
      *
+     * @param excludedAuthors RSS entries for given authors will be excluded
+     *     from the generated feed
+     *
      * @return Aggregated RSS feed for approved blog posts ordered by publication date
      */
     @Cacheable
-    public SyndFeed getRss(String feedUrl, int limit) {
+    public SyndFeed getRss(String feedUrl, int limit, Set<String> excludedAuthors) {
 
         Preconditions.checkArgument(
             StringUtils.isNotBlank(feedUrl), "feedUrl parameter cannot be blank");
 
-        final StopWatch stopWatch = new StopWatch();
-        log.debug("Building aggregated RSS feed...");
-        stopWatch.start();
+        StopWatch stopWatch = null;
+        if (log.isDebugEnabled()) {
+            stopWatch = new StopWatch();
+            log.debug("Building aggregated RSS feed...");
+            stopWatch.start();
+        }
 
         final Pageable pageRequest = new PageRequest(0, limit > 0 ? limit : Integer.MAX_VALUE);
         final List<BlogPost> approvedPosts =
-            blogPostRepository.findByApprovedTrueOrderByPublishedDateDesc(pageRequest);
+            blogPostRepository.findByApprovedTrueAndBlogAuthorNotInOrderByPublishedDateDesc(
+                pageRequest, firstNonNull(excludedAuthors, EMPTY_AUTHORS_SET)
```

Comment:
Would be better if you'd move `null` to `Controller`. This way we would avoid passing `null` through the application layers.